test/unit/proxy-alias-generators.test.ts

```
@@ -26,18 +26,78 @@ describe('proxy alias - generated runtime code (#814)', () => {
       )
       expect(out.pathname).toBe('/_scripts/p/eu.i.posthog.com/e/')
     })
+
+    it.each(['constructor', 'toString', '__proto__'])(
+      'treats inherited property name %s as an unaliased host',
+      (host) => {
+        // eslint-disable-next-line no-eval
+        const resolveUrl = eval(`(${generatePartytownResolveUrl('/_scripts/p')})`)
```

Biome (2.5.3)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)